### PR TITLE
Extensions: add micro:bit extension

### PR DESCRIPTION
**Pull request by microbit-mark**
cc @microbit-matt-hillsdon @jaustin 

I have not added this to the gallery as it doesn't fit into a particular category, but we can do if we feel it's helpful.

We might still want to update the name in pxt.json before merging

### CODE REVIEW

File: targetconfig.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "packages": {
         "approvedRepos": [
+            "pxt-microbit-v2-extension",
```
Comment (abchatra) [inline targetconfig.json:4]: This is incorrect. Needs the full path.

**Review by abchatra — CHANGES_REQUESTED** [review]

File: targetconfig.json (hunk 1 above)
Comment (microbit-mark) [reply targetconfig.json]: Whoops. Long week :). Done in https://github.com/microsoft/pxt-microbit/pull/3445/commits/df1f1f814f0a88de76f232cd1fe419a6e3b79191

**Comment by abchatra** [comment]
Extension also requires png.

**Comment by microbit-mark** [comment]
> Extension also requires png.

@abchatra Does https://github.com/microbit-foundation/pxt-microbit-v2-extension/blob/master/icon.png not cover this?

**Comment by abchatra** [comment]
It does. Sorry somehow missed it. Can we have v2 marked in big in that microbit png?

**Review by abchatra — APPROVED** [review]

**Pull request merged**